roles/dns/tasks/main.yml

```diff
@@ -27,3 +27,7 @@
   with_subelements:
   - "{{ dns_records_add | default({}) }}"
   - entries
+  register: nsupdate_result
+  until: nsupdate_result.dns_rc == 0
```

Comment:
when I tried `rc`, it returned "This structure does not contain `rc`." When I looked at the dictionary, the only thing resembling `rc` was the `dns_rc`. If it's supposed to look at something else, I don't know.